spring-cloud-dataflow-server-core/src/test/java/org/springframework/cloud/dataflow/server/service/impl/TaskServiceUtilsTests.java

```diff
@@ -362,6 +364,54 @@ public void testUseUserAccessTokenFromTaskPropsDefault() {
 		assertFalse("Use user access token should be false", result);
 	}
 
+	@Test
+	public void testConvertCommandLineArgsToCTRFormat() {
+		validateSingleCTRArgs("app.a.0=foo=bar", "--composed-task-app-arguments.base64_YXBwLmEuMA=foo=bar");
+		validateSingleCTRArgs("app.a.0=foo", "--composed-task-app-arguments.base64_YXBwLmEuMA=foo");
+		validateSingleCTRArgs("app.foo.bar", "--composed-task-app-arguments.app.foo.bar");
+		validateSingleCTRArgs("foo=bar", "foo=bar");
+	}
+
+	@Test
+	public void testConvertCommandLineArgsToCTRFormatWithNull() {
+		IllegalArgumentException thrown = Assertions.assertThrows(IllegalArgumentException.class,
```

Comment:
**[NIT]** This does not have to be done in this PR (or ever really) but... I recently discovered a newer form of "assert throws" in `assertThatXYZ().isThrownBy()...` which really simplifies these types of tests. In this case, we can simplify it down to something like:

```java
assertThatIllegalArgumentException().isThrownBy(() -> 
		TaskServiceUtils.convertCommandLineArgsToCTRFormat(Collections.singletonList(null)))
		.withMessage("Command line Arguments for ComposedTaskRunner contain a null entry.");
```
Again, not critical, just sharing what I have found.